mathparse/mathwords.py

```diff
@@ -365,6 +365,59 @@
             'bilhão': 1000000000,
             'trilhão': 1000000000000
         }
+    },
+    'THA': {
+        'unary_operators': {
+            'ยกกำลังสอง': '^ 2',
+            'ยกกำลังสาม': '^ 3',
+            'สแควรูท': 'sqrt'
+        },
+        'binary_operators': {
+            'บวก': '+',
+            'หาร': '/',
+            'ลบ': '-',
+            'คูณ': '*',
+            'ยกกำลัง': '^'
+        },
+        'numbers': {
+            'ศูนย์': 0,
+            'หนึ่ง': 1,
+            'สอง': 2,
+            'สาม': 3,
+            'สี่': 4,
+            'ห้า': 5,
+            'หก': 6,
+            'เจ็ด': 7,
+            'แปด': 8,
+            'เก้า': 9,
+            'สิบ': 10,
+            'สิบเอ็ด': 11,
+            'สิบสอง': 12,
+            'สิบสาม': 13,
+            'สิบสี่': 14,
+            'สิบห้า': 15,
+            'สิบหก': 16,
+            'สิบเจ็ด': 17,
+            'สิบแปด': 18,
+            'สิบเก้า': 19,
+            'ยี่สิบ': 20,
+            'สามสิบ': 30,
+            'สี่สิบ': 40,
+            'ห้าสิบ': 50,
+            'หกสิบ': 60,
+            'เจ็ดสิบ': 70,
+            'แปดสิบ': 80,
+            'เก้าสิบ': 90
+        },
+        'scales': {
+            'ร้อย': 100,
+            'พัน': 1000,
+            'หมื่น': 10000,
+            'แสน':100000,
```

Comment:
```suggestion
            'แสน': 100000,
```
A space needs to be added after this `:`.